### DISCUSSION TITLE
Spotube 3.0 doesn't start on Linux MInt 20.3

### DISCUSSION THREAD

**Discussion by cirodimaio** [KRTirtho/spotube#564] [Q&A]
Spotube Appimage 3.0 doesn't start on Linux MInt 20.3 from terminal gives me this error ```
```
spotube
spotube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by spotube)
```

**Comment by KRTirtho** [comment]
Unfortunately, Linux Mint 20.3 is based on Ubuntu 20.04. But spotube's .deb package is supported by Ubuntu >= 22.04. So please use the Flatpak version. Or you can build from source
